runtime/ftplugin/man.vim

```diff
@@ -143,32 +138,32 @@ func <SID>GetPage(...)
       endwhile
     endif
     if &filetype != "man"
-      new
-      setl nonu fdc=0
+      tabnew
+      setlocal nonumber foldcolumn=0
     endif
   endif
   silent exec "edit $HOME/".page.".".sect."~"
   " Avoid warning for editing the dummy file twice
-  setl buftype=nofile noswapfile
+  setlocal buftype=nofile noswapfile
 
-  setl ma nonu nornu nofen
-  silent exec "norm 1GdG"
+  setlocal modifiable nonumber norelativenumber nofoldenable
+  silent norm! 1GdG
   let $MANWIDTH = winwidth(0)
   silent exec "r!/usr/bin/man ".s:GetCmdArg(sect, page)." | col -b"
   " Remove blank lines from top and bottom.
   while getline(1) =~ '^\s*$'
-    silent norm ggdd
+    silent norm! ggdd
   endwhile
   while getline('$') =~ '^\s*$'
-    silent norm Gdd
+    silent norm! Gdd
   endwhile
   1
-  setl ft=man nomod
-  setl bufhidden=hide
-  setl nobuflisted
+  setlocal ft=man nomodified nomodifiable readonly bufhidden=hide nobuflisted
+  setlocal tabstop=8 nolist colorcolumn=0
+  nnoremap <nowait><buffer> q <c-w>c
```

Comment:
I'm a little iffy on this. My hunch is that users are used to pressing `q` to exit `man` after `K`, so we should preserve that behavior.